app/models/cip.rb

```diff
@@ -0,0 +1,4 @@
+# frozen_string_literal: true
+
+class Cip < ApplicationRecord
```

Comment:
@kicferk1 does this conflict with what you've just done?

Reply:
Eh, maybe. Technically I think a CIP would be two CourseYears, which is the highest level of content I've done. I think we need to add a card to reconcile and connect CIP content to cohorts and lead providers anyway, to one of the trello board. Wanna do it?

Reply:
Using an abbreviation / acronym for a core entity is probably not a great idea. Can we just use the full name?